tests/SalesOrdersClientTest.php

```diff
@@ -45,6 +47,49 @@ public function testCreateAndActivateOrder()
         );
     }
 
+    public function testCreateAndActivateTransferAppOrder()
+    {
+        // Setup the client
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new SalesOrdersClient($environment);
+
+        // Create the request
+        $req = new CreateAndActivateOrderRequest();
+
+        // Create the line items
+        $gSuite = SalesOrdersUtils::buildLineItem('MP-7TMH5K8N7KNNRZ5NC4RNNS2JFK64HMNC');
+        $lineItems = array($gSuite);
+
+        // Create the custom field
+        $gSuiteCustomField = new CustomField();
+        $gSuiteCustomField->setProductId('MP-7TMH5K8N7KNNRZ5NC4RNNS2JFK64HMNC');
+
+        $domain = SalesOrdersUtils::buildField('domain', 'Domain Name', "domain.com", FieldType::TEXT,);
+        $transferToken = SalesOrdersUtils::buildField('transfer_token',  'Transfer Token', "TOKEN123TOKEN", FieldType::TEXT,);
+        $autoTransfer = SalesOrdersUtils::buildField('auto_transfer', 'Automatic Transfer', "true", FieldType::CHECKBOX);
```

Comment:
Normally the transfer workflow requires an additional step of "Approving" the transfer once the transfer app is activated. By supply `true` to the `auto_transfer` field in the order form, it will skip that additional approval step and automatically do the transfer.